assignments/objects.js

```diff
@@ -10,35 +10,106 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+const interns = [{
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"},
+  {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"},{
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"},{
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"},{
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"},]
 
 // Write your intern objects here:
 
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
-// Mitzi's name
-
+  // Mitzi's name
+console.log(interns[0].name)
 // Kennan's ID
-
+console.log(interns[1].id)
 // Keven's email
-
+console.log(interns[2].email)
 // Gannie's name
-
+console.log(interns[3].name)
 // Antonietta's Gender
+console.log(interns[4].gender)
 
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
+const interns = [{
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"},
+  {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak":() => console.log("Hello, my name is Kennan!")
+  },{
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"},{
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"},{
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"},]
+
+console.log(interns[1].speak())
```

Comment:
here since you used an array for your interns and the challenge is telling you the name of the intern you can use the `.findIndex()` method to to find the index of the object that is hosting that name and add the function via a leftdot. This way you can add the function without having to touch your array

ex
```
let index = interns.findIndex(obj => obj.name === 'Kennan');

interns[index].speak = () => {
  console.log(`hi my Name is Gannie`)
}
```